jina/orchestrate/flow/base.py

```diff
@@ -1961,10 +1961,12 @@ async def _f():
                 f'{self.num_deployments} Deployments (i.e. {self.num_pods} Pods) are running in this Flow'
             )
 
-            print(
-                'Do you love Open Source? Help us get better and be heard in just 1 minute and 30 seconds :sparkling_heart:'
-                'Your feedback will help us build better features for [link=https://github.com/jina-ai/jina]Jina[/link], your loved open-source project :tada: https://10sw1tcpld4.typeform.com/jinasurveyfeb23?utm_source=jina Take the Jina user survey!'
-            )
+            if 'JINA_HIDE_SURVEY' not in os.environ:
+                print(
+                    'Do you love Open Source? Help us get better and be heard in just 1 minute and 30 seconds :sparkling_heart: '
+                    'Your feedback will help us build better features for [link=https://github.com/jina-ai/jina]Jina[/link], your loved open-source project :tada: https://10sw1tcpld4.typeform.com/jinasurveyfeb23?utm_source=jina Take the Jina user survey! '
```

Comment:
```suggestion
                    'Do you love open source? Help us improve [link=https://github.com/jina-ai/jina]Jina[/link] in just 1 minute and 30 seconds by taking our survey: https://10sw1tcpld4.typeform.com/jinasurveyfeb23?utm_source=jina'
```

I'm cutting the wordage and bs to make less annoying